94-appendixD.Rmd

```diff
@@ -500,7 +500,7 @@ with?
 
 **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What can we say about the distribution of `gain`?  Describe it in a few sentences using the plot and the `gain_summary` data frame values.
 
-**Solution**: Most of the time the gain is a little under zero, most of the time the gain is between -50 and 50 minutes. There are some extreme cases however!
+**Solution**: Most of the time the gain is a little above zero (the median is 7, meaning gain is above 0 at least 50% of the time), most of the time the gain is between -50 and 50 minutes. There are some extreme cases however!
```

Comment:
This sentence might need some cleaning up since "most of the time" is repeated twice with two different situations.